base/errorshow.jl

```diff
@@ -367,6 +367,12 @@ function showerror(io::IO, ex::MethodError)
     nothing
 end
 
+function showerror(io::IO, exc::MemberAccessError)
+    @nospecialize
+    println(io, "MemberAccessError : member access of this field on this DataType is not entertained.\n")
```

Comment:
```suggestion
    println(io, "UndefFieldError: type $(exc.type) has not field $(exc.field)")
```
- Member is not a term in Julia
- The trailing "\n" is handled by `println`.
- "does not have field" is more specific and clear than "access is not entertained"